README.md

```diff
@@ -1 +1,78 @@
-# tic_tac_toe_ruby
\ No newline at end of file
+# tic_tac_toe_ruby
+> Building Tic Tac Toe game in Ruby
+ In this repository, we will show you the source code of this game on Ruby programming language, how to run it and explain some related concepts.
+
+ bin
+ Within the bin/ directory we generally put code that relates to running our actual program, running CLI applications Executable files are any files that contain instructions in a form that a computer's operating system or application can understand and follow. 
+
```

Comment:
Try to personalize the content:
'The `/bin` directory contains code that relates to running our actual program.....'